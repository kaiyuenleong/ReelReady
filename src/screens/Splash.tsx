import React from "react";
import { Text, View } from "react-native";
import { Gradient } from "../components/common";

const Splash: React.FC = () => {
  return (
    <Gradient>
      <View>
        <Text>Splash Screen</Text>
      </View>
    </Gradient>
  )
}

export default Splash;