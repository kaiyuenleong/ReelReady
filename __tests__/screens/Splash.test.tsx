import React from "react";
import renderer from "react-test-renderer";

import Splash from "../../src/screens/Splash";

describe('<Splash />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Splash />).toJSON();
    expect(tree).toMatchSnapshot();
  })
})