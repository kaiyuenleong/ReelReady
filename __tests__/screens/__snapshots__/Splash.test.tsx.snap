// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Splash /> renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <ViewManagerAdapter_ExpoLinearGradient
    proxiedProperties={
      Object {
        "colors": Array [
          4289198296,
          4283231449,
          4279833944,
          4279304248,
        ],
        "endPoint": undefined,
        "locations": Array [
          0,
          0.1,
          0.8,
          1,
        ],
        "startPoint": undefined,
      }
    }
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <Image
        resizeMode="contain"
        source={1}
        style={
          Object {
            "height": 250,
            "width": 250,
          }
        }
      />
    </View>
  </ViewManagerAdapter_ExpoLinearGradient>
</View>
`;
